packages/docusaurus-tldraw-plugin/src/tldr.jsx

```diff
@@ -0,0 +1,38 @@
+import BrowserOnly from '@docusaurus/BrowserOnly';
+import 'tldraw/tldraw.css';
+import { useMemo } from 'react';
+import { Tldraw, parseTldrawJsonFile, createTLSchema } from 'tldraw';
+
+export const Tldr = ({ children, height }) => {
+	const { value: store, err } = useMemo(() =>
+		parseTldrawJsonFile({ json: children, schema: createTLSchema() }),
+		[children]
+	);
+
+	if (err) {
+		console.log(result.err);
+		return <div>Error loading tldr file</div>;
+	}
+	else if(!store) {
```

Comment:
drop the `else` here ... at least this is not necessary and I thought would give linting error?